package.json

```diff
@@ -4,6 +4,7 @@
   "version": "3.0.141",
   "author": "OSCAFRICA",
   "dependencies": {
+    "dotenv": "8.2.0",
```

Comment:
Hey, I think this should be a `devDependency`

Reply:
It's perfectly fine to use `dotenv` in production.

Reply:
I noticed you only used it for the integration tests, so it's not needed in the main project at all

Reply:
This is very necessary. Especially during `CI & CD` and storing secrets within `env variables`.

Reply:
I believe what @ashinzekene is trying to hint at is:

* The `NODE_ENV` for your testing environment wouldn't be `production` (so technically it's still a dev dependency)
* In production, you won't need this because we currently only pull environment variables for pulling variables needed to run the tests (we don't use environment variables for the gatsby application itself) - we have a `.env.example` file yet so it'd be great if your PR could add that.